frontend/src/routes/manage/Series/index.tsx

```diff
@@ -0,0 +1,174 @@
+                    title
+                    created
+                    updated
+                    syncedData { description }
+                    entries {
+                        __typename
+                        ...on AuthorizedEvent {
+                            isLive
+                            syncedData { thumbnail audioOnly }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+export type SeriesConnection = NonNullable<SeriesManageQuery$data["currentUser"]>["mySeries"];
+export type Series = SeriesConnection["items"];
+export type SingleSeries = Series[number];
+export type Entry = SingleSeries["entries"][number];
+
+export const seriesColumns: ColumnProps[] = [
+    {
+        key: "EVENT_COUNT",
+        label: "manage.my-series.content",
+        headerWidth: 112,
+        column: series => <td css={{ fontSize: 14 }}>
+            {i18n.t("manage.my-series.no-of-videos", { count: series.entries?.length })}
+        </td>,
+    },
+    {
+        key: "UPDATED",
+        label: "manage.item-table.columns.updated",
+        column: series => <DateColumn date={series.updated} />,
+    },
+    {
+        key: "CREATED",
+        label: "manage.item-table.columns.created",
+        column: series => <DateColumn date={series.created} />,
+    },
+];
+
+
+export const SeriesRow: React.FC<{ series: SingleSeries }> = ({ series }) => {
+    // Todo: change to "series details" route when available
+    const link = DirectSeriesRoute.url({ seriesId: series.id });
+
+    // Seems odd, but simply checking `e => e.__typename === "AuthorizedEvent"` will produce
+    // TS2339 errors when compiling.
+    type AuthorizedEvent = Extract<Entry, { __typename: "AuthorizedEvent" }>;
+    const isAuthorizedEvent = (e: Entry): e is AuthorizedEvent =>
+        e.__typename === "AuthorizedEvent";
+
+    const thumbnails = series.entries
+        .filter(isAuthorizedEvent)
+        .map(e => ({
+            isLive: e.isLive,
+            audioOnly: e.syncedData ? e.syncedData.audioOnly : false,
+            thumbnail: e.syncedData?.thumbnail,
+        }));
+
+    return (
+        <TableRow
+            thumbnail={<Link to={link} css={{ ...thumbnailLinkStyle }}>
+                <span css={{ "> div": { width: "100%" } }}>
+                    <ThumbnailStack title={series.title} {...{ thumbnails }} />
+                </span>
+            </Link>}
+            title={<Link to={link} css={{ ...titleLinkStyle }}>{series.title}</Link>}
+            description={series.syncedData && <SmallDescription
+                css={{ ...descriptionStyle }}
+                text={series.syncedData.description}
+            />}
+            syncInfo={{
+                isSynced: !!series.syncedData,
+                notReadyLabel: "series.not-ready.label",
+            }}
+            customColumns={seriesColumns.map(col => (
+                <Fragment key={col.key}>{col.column(series)}</Fragment>
```

Comment:
This should rather be `<col.column ...>` and changing the `column` type to `({ item: ... }) => ReactNode`. Or `ComponentType<{ item: ... }>`. The point is that we want to render it using the normal JSX react magic. Sometimes its useful to have a simple function that returns JSX and to call it directly. But things can go wrong, mostly related to hooks.
For example, in some columns `<DateColumn>` is used, which in turn uses many hooks. When not rendering with JSX, but calling a function, these hooks are counted as part of the parent component, which leads to the typical problems with hooks: they always need to be called in the exact same order for each component, unconditionally. Apparently this doesn't lead to crash here but it's close. So yeah, treat `column` as a component.